united-states-international-trade-commission.csl

```diff
@@ -126,9 +130,12 @@
   <macro name="contributors-note">
     <group delimiter=" ">
       <choose>
-        <if match="any" variable="note">
-          <text variable="note"/>
+        <if match="all" variable="note" type="speech">
+          <text value="Warning: the 'extra' field does not populate correctly with presentation citation item types. Please edit this citation item in the Zotero app to resolve. A manual overwrite may be needed for abbreviated names." strip-periods="false" font-weight="normal"/>
```

Comment:
what's going on there? That seems like something that should get fixed, maybe at Zotero?